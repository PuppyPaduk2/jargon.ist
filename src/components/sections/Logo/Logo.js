// @flow

import React from 'react';
import cx from 'classnames';

import logo from '../../../img/logo.svg';

import './Logo.scss';

type Props = {
  className?: string,
};

const Logo = ({ className }: Props) => (
  <span className={cx('c-logo', className)}>
    <img src={logo} alt="Jargon.ist" height="24" className="u-img-responsive" />
  </span>
);

Logo.defaultProps = {
  className: '',
};

export default Logo;
