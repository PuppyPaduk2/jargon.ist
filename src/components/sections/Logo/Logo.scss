@import '../../../styles/abstracts/index';

.c-logo {
  align-items: center;
  display: flex;
  flex-direction: row;

  &__text {
    font-size: px-to-rem(24px);
    line-height: px-to-rem(24px * $g-line-height-base);
    padding-left: pad(xsmall);
  }
}
