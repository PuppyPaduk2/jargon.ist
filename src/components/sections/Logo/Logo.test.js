// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Logo from './Logo';

it('renders correctly', () => {
  const wrapper = shallow(<Logo />);

  expect(wrapper).toMatchSnapshot();
});
