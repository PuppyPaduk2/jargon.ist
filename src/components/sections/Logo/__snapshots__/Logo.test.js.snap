// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="c-logo"
>
  <img
    alt="Jargon.ist"
    className="u-img-responsive"
    height="24"
    src={
      Object {
        "process": [Function],
      }
    }
  />
</span>
`;
